package.json

```diff
@@ -67,8 +67,9 @@
     "prettier-plugin-curly": "^0.2.1",
     "prettier-plugin-packagejson": "^2.5.0",
     "rimraf": "^5.0.7",
-    "turbo": "^1.13.3",
+    "turbo": "^2.0.3",
     "typescript": "^5.4.5",
     "yargs": "^17.7.2"
-  }
+  },
+  "packageManager": "bun@1.1.12"
```

Comment:
does this actually work?

Reply:
This entire commit is the result of running `npx @turbo/codemod migrate` ... And all turbo commands continue to work. Removing the `packageManager` makes turbo bark at me.
<img width="359" alt="Screenshot 2024-06-08 at 12 33 15" src="https://github.com/ixahmedxi/orbitkit/assets/52371939/59b6d557-863f-4707-9149-b2a2a918b271">

Reply:
ohhh makes sense then.